client/src/components/Form/Form.js

```diff
@@ -0,0 +1,302 @@
+import React, { useState } from "react";
+
+const Form = () => {
+  const [teamName, setTeamName] = useState("");
+  const [projectBrief, setProjectBrief] = useState("");
+  const [githubRepo, setGithubRepo] = useState("");
+  const [githubProjectBoard, setGithubProjectBoard] = useState("");
+  const [deployedApp, setDeployedApp] = useState("");
+  const [slackChannel, setSlackChannel] = useState("");
+
+  const [memberName1, setMemberName1] = useState("");
+  const [memberName2, setMemberName2] = useState("");
+  const [memberName3, setMemberName3] = useState("");
+  const [memberName4, setMemberName4] = useState("");
+
+  const [role1, setRole1] = useState("");
+  const [role2, setRole2] = useState("");
+  const [role3, setRole3] = useState("");
+  const [role4, setRole4] = useState("");
+
+  const [githubUsername1, setGithubUsername1] = useState("");
+  const [githubUsername2, setGithubUsername2] = useState("");
+  const [githubUsername3, setGithubUsername3] = useState("");
+  const [githubUsername4, setGithubUsername4] = useState("");
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+
+    try {
+      const response = await fetch("http://localhost:8000/submit-form", {
+        method: "POST",
+        headers: {
+          "Content-Type": "application/json",
+        },
+        body: JSON.stringify({
+          teamName,
+          projectBrief,
+          githubRepo,
+          githubProjectBoard,
+          deployedApp,
+          slackChannel,
+
+          memberName1,
+          memberName2,
+          memberName3,
+          memberName4,
+          role1,
+          role2,
+          role3,
+          role4,
+          githubUsername1,
+          githubUsername2,
+          githubUsername3,
+          githubUsername4,
+        }),
+      });
+
+      const data = await response.json();
+      console.log(data);
+
+      setTeamName("");
+      setProjectBrief("");
+      setGithubRepo("");
+      setGithubProjectBoard("");
+      setDeployedApp("");
+      setSlackChannel("");
+
+      setMemberName1("");
+      setMemberName2("");
+      setMemberName3("");
+      setMemberName4("");
+
+      setRole1("");
+      setRole2("");
+      setRole3("");
+      setRole4("");
+
+      setGithubUsername1("");
+      setGithubUsername2("");
+      setGithubUsername3("");
+      setGithubUsername4("");
+    } catch (error) {
+      console.error("Error submitting form:", error);
+    }
+  };
+
+  return (
+    <form onSubmit={handleSubmit}>
+      <div>
+        <label htmlFor="teamName">Team Name:</label>
+        <input
+          type="text"
+          id="teamName"
+          name="teamName"
+          value={teamName}
+          onChange={(e) => setTeamName(e.target.value)}
+          required
+        />
+      </div>
+      <div>
+        <label htmlFor="projectBrief">Project Brief:</label>
+        <textarea
```

Comment:
you can change this `textarea` into an input as it will only be accepting the url of the project brief